decorater.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'nameable'
+require_relative 'person'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.capitalize
+  end
+end
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name[0..9]
+  end
+end
+
+person = Person.new(1, 22, 'maximilianus')
+person.correct_name
+
+capitalized_person = CapitalizeDecorator.new(person)
+puts capitalized_person.correct_name
+capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
+puts capitalized_trimmed_person.correct_name
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you remove these console commands to make your code clean and professional 👍